eos-prune-printers.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Endless prune printers
+After=cups.service
+ConditionPathExists=!/var/eos-printers-pruned
+ConditionKernelCommandLine=!endless.live_boot
+
+[Service]
+Type=oneshot
+ExecStart=/usr/sbin/eos-prune-printers
+StandardOutput=journal+console
```

Comment:
Since this isn't an early boot service, I think just the default of `journal` is fine.

Reply:
Sure